scripts/discover/as-domains-from-domain-ssl-certs

```diff
@@ -23,14 +23,13 @@ find recon/domains/ -maxdepth 1 -type d  | tail -n +2 | while read domain_path;
   if [ -f "$domain_path/ssl-cert-domains.txt" ]; then
     continue
   fi
-  domain=$(echo "$domain_path" | cut -d/ -f3)
+  root_domain=$(basename "$domain_path")
```

Comment:
i seem to remember basename not being installed by default on a lot of systems. we should add this to the dependencies.